scripts/updateUsers.ts

```
@@ -112,9 +112,33 @@ function buildUserDataMap(fetched: UsersData[]): Map<string, {
         bestShare: parseFloat(workerData.bestshare),
         bestEver: BigInt(workerData.bestever).toString(),
         updatedAt: new Date(),
-      });
+      };
+
+      const existingWorker = workersMap.get(workerName);
+      if (existingWorker) {
+        existingWorker.hashrate1m = (BigInt(existingWorker.hashrate1m) + BigInt(newWorker.hashrate1m)).toString();
```

ESLint

[error] Replace `BigInt(existingWorker.hashrate1m)·+·BigInt(newWorker.hashrate1m)` with `⏎··········BigInt(existingWorker.hashrate1m)·+·BigInt(newWorker.hashrate1m)⏎········`

(prettier/prettier)